docker/Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM scientificlinux/sl:7
+FROM fermilab/fnal-wn-sl7
```

Comment:
on Fermilab redmine on Singularity 
"When building a custom Singularity images, we recommend to base the image on the FNAL WN image.
This will make sure the image has all tools required to interact with Fermilab infrastructure (dCache, SAM, ...)."

Reply:
Agree.